### DISCUSSION TITLE
do NOT modify the key name in the UI

### DISCUSSION THREAD

**Discussion by tacman** [meilisearch/product#781] [Feedback & Feature Proposal]
Beside the inability to search by a set of primary keys, my other least favorite thing of the otherwise awesome meilisearch is that the key names are modified when using the UI to display a record

![image](https://github.com/user-attachments/assets/a125d629-0377-4032-8494-80d7c34757d3)

Why on earth would I want to see anything but my exact keys.  Removing the underscores and changing everything to uppercase does not help when I'm debugging.  PLEASE simply show the _exact_ keys that were sent when index (e.g. obj_count, lastScrapedItemNumber, _label)

If keys with diacriticals (accents) or hyphens aren't supported, that should fail at the indexing level (as I believe they are) and _not_ modified.

Thanks.  It should be a tiny change, simply removing the key name display change.    The UI should be a reflection of the underlying data.

![image](https://github.com/user-attachments/assets/f3f72836-0536-4ec6-860b-1b6f2646d62c)

**Comment by curquiza** [comment]
Hello @tacman 

We have a good first issue opened here to fix this problem: https://github.com/meilisearch/mini-dashboard/issues/494

We are a small team so we cannot spend time on it, however, we are available to review any PR to fix it. I will do a release right after once the PR is merged if anyone of the community wants to spend time on it

Thank you!

**Reply by curquiza** [reply]
Will be fixed in v1.11, thank you for your PR @tacman
